src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -214,6 +269,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('link_rewrite', TranslatableType::class, [
+                'label' => $this->trans('Friendly URL', 'Admin.Global'),
+                'help' => $this->trans('Only letters, numbers, underscore (_) and the minus (-) character are allowed.', 'Admin.Catalog.Help'),
                 'type' => TextType::class,
```

Comment:
The help text here is only correct if "Accented URL" (in Traffic & SEO) is turned off. I think we should have either 

1. 2 different help texts depending on if the option turned on or not. 
2. Modify help text to let user know about the setting and what it changes.

I like the second option more as it would make it clear that Accented URL's changes the way this field is validated.

Reply:
What do you think of the following wording? (I followed your second idea)

Unless the 'Accented URL' option is enabled (in Shop parameters > Traffic & SEO), only letters, numbers, underscores (_), and hyphens (-) are allowed.

Reply:
Makes sense to me. Added your suggestion.